src/vue-components/views/gridEditView.vue

```diff
@@ -13,24 +13,7 @@
                 <button tabindex="32" @click="redo"  :aria-label="$t('redo')" :disabled="!canRedo || doingUndoRedo" class="small spaced"><i class="fas fa-redo"></i> <span class="hide-mobile">{{ $t('redo') }}</span></button>
             </div>
         </header>
-        <div>
-            <edit-element v-if="showEditModal" v-bind:edit-element-id-param="editElementId" :grid-instance="getGridInstance()" :grid-data-id="gridData.id" @close="showEditModal = false" @mark="markElement" @actions="(id) => {editElementId = id; showActionsModal = true}"/>
-        </div>
-        <div>
-            <add-multiple-modal v-if="showMultipleModal" v-bind:grid-data="gridData" :grid-instance="getGridInstance()" @close="showMultipleModal = false"/>
-        </div>
-        <div>
-            <grid-dimension-modal v-if="showDimensionsModal" v-bind:grid-data-param="gridData" :is-global-grid="metadata.globalGridId === gridData.id" @close="showDimensionsModal = false" @save="setDimensions"/>
```

Comment:
before: i can see which props are passed to `grid-dimension-modal` and what happens on the emitted events from this modal within this single line.

afterwards: in order to see which props are passed I have to open `grid-dimension-modal` and examine the `computed` properties there, where some of them are the equivalent to the passed params. But I don't know where to find to place, where the params are passed. After some minutes I find that it happens at `case CONTEXT_GRID_DIMENSIONS: {` below in this file. However the code within this `case` statement is now 6 lines long instead of 1 before. And I still have to examine `store.js` and understand how the vuex store works in order to fully understand the passing of the parameters. In `handleModalSave` below I now have some "if/else" to decide from which modal we came and do things differently based on this info, so in some way it's undoing the generalization which is done by importing a single `component` instead of multiple modal components.

So in total I think much complexity was added for winning some cleaner looking code here only having one `component` instead of multiple modals. I'm sorry because I know it was much work, but I don't like this new way of passing parameters and handling events from the modals, since the logic is now spread to much more places and it's much harder to understand. And there are enough places within AG which are hard to understand, so every big change should make things easier and more clear.